io.github.sigmasd.VirtAudio.yml

```diff
@@ -0,0 +1,265 @@
+app-id: io.github.sigmasd.VirtAudio
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: virtaudio
+finish-args:
+  # Display
+  # wayaland is currently broken in slint, but good news its going to be fixed in 1.9.3 (already fixed in master)
+  - --socket=x11
+  - --share=ipc
+  # Audio access
+  - --socket=pulseaudio
+  # Network access for local server and WebSocket connections
+  - --share=network
+  # Needed for mesa
+  - --device=dri
+  # Allow creating temporary files (for creating virtual mics)
+  - --filesystem=/tmp:create
+modules:
+  # needed for libinput
+  - name: libevdev
+    buildsystem: meson
+    config-opts:
+      - -Dtests=disabled
+      - -Ddocumentation=disabled
+    sources:
+      - type: archive
+        url: https://www.freedesktop.org/software/libevdev/libevdev-1.13.3.tar.xz
+        sha256: abf1aace86208eebdd5d3550ffded4c8d73bb405b796d51c389c9d0604cbcfbf
+        x-checker-data:
+          type: anitya
+          project-id: 20540
+          stable-only: true
+          url-template: https://www.freedesktop.org/software/libevdev/libevdev-$version.tar.xz
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share/man
```

Comment:
add these to top level cleanup `*.a` too. no need to repeat.

Reply:
done